packages/chatbot-server-mongodb-public/src/config.ts

```diff
@@ -41,6 +43,9 @@ export const {
   OPENAI_CHAT_COMPLETION_DEPLOYMENT,
 } = assertEnvVars(CORE_ENV_VARS);
 
+const VERIFIED_ANSWERS_CONNECTION_NAME =
+  process.env.VERIFIED_ANSWERS_CONNECTION_NAME ?? "verified_answers";
```

Comment:
```suggestion
const VERIFIED_ANSWERS_COLLECTION_NAME =
  process.env.VERIFIED_ANSWERS_COLLECTION_NAME ?? "verified_answers";
```

Reply:
also, i don't think we should do the optionality here since it's our implementation. either it's provided or it isn't and we throw an err.

Reply:
also per below comment, i think we can include a default collection name in the `MongoDBVerifiedAnswerStore.ts` file, removing the need for declaring collection name here.